pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -179,17 +182,18 @@ func (r *ReconcileMachineHealthCheck) Reconcile(ctx context.Context, request rec
 
 	// health check all targets and reconcile mhc status
 	currentHealthy, needRemediationTargets, nextCheckTimes, errList := r.healthCheckTargets(targets, mhc.Spec.NodeStartupTimeout.Duration)
-	mhc.Status.CurrentHealthy = &currentHealthy
+	healthyCount := len(currentHealthy)
+	mhc.Status.CurrentHealthy = &healthyCount
```

Comment:
We look to be changing the meaning/contents of a status field.  Can you please reach out to the cloud team for the rules about if/when/how API breaking changes are permitted?

Reply:
The change was not in status field but in the return value of healthCheckTargets method.
Instead of returning a count(number) of healthy targets it returns a slice of the actual objects.
The reason for this change is because those elements are needed later as an input for cleanEMR method.